public/locales/en-US/send.ftl

```diff
@@ -80,3 +84,6 @@ footerLinkAbout = About Test Pilot
 footerLinkPrivacy = Privacy
 footerLinkTerms = Terms
 footerLinkCookies = Cookies
+requirePassword = Require a password to download this file
```

Comment:
Can you use a string ID that describes better the role of this string, or add a localization comment to explain the context?